### PR TITLE
FKM-9. Wrong file usage count

### PR DESCRIPTION
**Pull request by dporplenko**

### CODE REVIEW

**Comment by ffw-bot-ua** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/24/

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/25/

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/27/

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/28/

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/29/

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/30/

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/31/

**Build Log**
*last 25 lines*

 ```
[...truncated 134 lines...]
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
changed: [localhost]

TASK: [Importing from local backup] ******************************************* 
failed: [localhost] => {"failed": true}
msg: ERROR 1114 (HY000) at line 17387: The table 'watchdog' is full


FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=20   changed=13   unreachable=0    failed=1   

Build step 'Execute shell' marked build as failure
```

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/32/

**Build Log**
*last 25 lines*

 ```
[...truncated 134 lines...]
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
changed: [localhost]

TASK: [Importing from local backup] ******************************************* 
failed: [localhost] => {"failed": true}
msg: ERROR 3 (HY000) at line 25: Error writing file './drupal32/actions.frm' (Errcode: 28)


FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=20   changed=13   unreachable=0    failed=1   

Build step 'Execute shell' marked build as failure
```

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/33/

File: drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install [hunk 1]
```diff
@@ -682,4 +682,9 @@ function favrskov_helper_update_7037() {
 
     file_usage_add($file, 'media', $file->entity_type, $file->entity_id);
   }
+
+  module_enable(array(
```
Comment (BR0kEN-) [inline drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install]: Shouldn't you add the same to `hook_update_N()`?

**Pull request merged**